src/services/reviews.js

```diff
@@ -260,7 +259,7 @@ module.exports = class reviewsHelper {
 
 			// Publish resource if isPublishResource is true
 			if (isPublishResource) {
-				const publishResource = await this.publishResource(resourceId, resource.user_id)
+				const publishResource = await this.publishResource(resourceId, resource.user_id, orgId)
```

Comment:
@priyanka-TL to remove here and everywhere else where org is not required and can be derived from the resource in the hook of the model?